src/main/webapp/script.js

```diff
@@ -24,3 +32,75 @@ function addRandomQuote() {
         quoteContainer.innerText = quote;
     });
 }
+
+/**
+ * Adds past journal entries to the page
+ */
+function addPastEntries() {
+  fetch('/my-data-url').then(response => response.json()).then((pastEntries) => {
+    // Convert message JSON array to HTML elements then add to page
+    const pastEntriesListElement = document.getElementById('past-entries-container');
+    for(var i = 0; i < pastEntries.length; i++) {
+          pastEntriesListElement.appendChild(createEntryElement(pastEntries[i]));
+        }
+  });
+}
+
+/** Creates a <div> element containing a journal entry. */
+function createEntryElement(entry) {
+  const entryElement = document.createElement('div');
+  entryElement.setAttribute("class", "entry");
+
+  const textElement = document.createElement('div');
+  textElement.setAttribute("class", "entry-text");
+  textElement.innerText = "Entry: " + entry.textEntry;
+
+  const moodElement = document.createElement('div');
+  moodElement.setAttribute("class", "entry-mood");
+  moodElement.innerText = "MoodValue: " + entry.moodValue;
+
+  const songElement = document.createElement('div');
+  songElement.setAttribute("class", "entry-song");
+  songElement.innerText = "Song: " + entry.songTitle + " by " + entry.artistName;
+
+  const timestampElement = document.createElement('div');
+  timestampElement.setAttribute("class", "entry-timestamp");
+  timestampElement.innerText = getTimeAgo(entry.timestamp);
```

Comment:
I suggest moving the timestamp section to be above the entry section. It might help users keep track of their entries better.